swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -243,77 +257,13 @@ Follow the instructions given in this section to develop the service.
 
     - Define the `hospitalEP` variable. Later, the relevant client is assigned to this variable based on the `hospital_id` value.
 
-        ```ballerina
-        http:Client hospitalEP;
-        ```
-
-    - Use a [typed binding pattern with a mapping binding pattern to destructure](https://ballerina.io/learn/by-example/rest-binding-pattern-in-mapping-binding-pattern/) the payload and assign required components of the value to separate variables.
-        
-        ```ballerina
-        ReservationRequest {hospital_id, patient, doctor, ...reservationRequest} = payload;
-        ```
-    
-        Here,
-        - The `hospital_id` value is assigned to the `hospital_id` variable. 
-        - The `patient` value is assigned to the `patient` variable of type `Patient` (fields are exactly those expected by the `Patient` record).
-        - The `doctor` value is assigned to the `doctor` variable of string type.
-        - The remaining components of the payload are collected in a mapping value and assigned to the `reservationRequest` variable.
-
-    - The `log` functions are used to [log](https://ballerina.io/learn/by-example/#log) information at `INFO`, `DEBUG`, and `ERROR` log levels.
-
     - Use a [match statement](https://ballerina.io/learn/by-example/match-statement) to assign the relevant client to the `hospitalEP` variable based on the `hospital_id` value.
 
-        ```ballerina
-            match hospital_id {
-                GRAND_OAK => {
-                    hospitalEP = grandOakEP;
-                }
-                CLEMENCY => {
-                    hospitalEP = clemencyEP;
-                }
-                _ => {
-                    hospitalEP = pineValleyEP;
-                }
-            }
-        ```
-
     - Make a `POST` request to the backend hospital service to make the reservation. The `category` value is used as a path parameter.
```

Comment:
- Make a `POST` request to the relevant backend hospital service to make the reservation. The `category` value is used as a path parameter.